scripts/build_test_pipeline.sh

```diff
@@ -28,4 +28,4 @@ chmod +x * && ./etherscan-mock.sh
 
 #Update config
 cd $SRC_DIR
-jq '.persistence.postgres.host="20.112.58.141"|.persistence.postgres.user="miningcore"|.persistence.postgres.database="miningcore"|.persistence.postgres.password="c#VB?8?h6R"' $OUT_DIR/config_test.json > tmp.$$.json && mv tmp.$$.json $OUT_DIR/config_test.json
+jq '.persistence.postgres.host="20.112.58.141"|.persistence.postgres.user="miningcore"|.persistence.postgres.database="miningcore"|.persistence.postgres.password="c#VB?8?h6R"|.persistence.cosmos.endpointUrl="https://pooltest.documents.azure.com:443/"|.persistence.cosmos.authorizationKey="SjLHfbJG7MLoSciwDJXVBRkPRAtkflTsDWwsictRJtPwYE2ILXXHqQWZFpdLzaq6cnEcIKP2FtSXnJIiDCv5ZQ=="' $OUT_DIR/config_test.json > tmp.$$.json && mv tmp.$$.json $OUT_DIR/config_test.json
```

Comment:
Added the below story to pull credentials from the key vault.
https://jira.corp.nortonlifelock.com/browse/CRBN-348